test cases/failing/39 kwarg assign/meson.build

```diff
@@ -0,0 +1,4 @@
+project('assign in kwarg', 'c')
+
+executable('prog', 'dummy.c', args = 'prog.c')
```

Comment:
Would be nice to be able to check how a test failed, perhaps by grepping the error message in the exception.